src/components/CookiesDialog.vue

```diff
@@ -0,0 +1,118 @@
+<!-- eslint-disable vue-i18n/no-raw-text -->
+<template>
+  <div
+    class="cookies-dialog"
+    @click.stop
+  >
+    <ButtonPlain
+      class="button-cancel"
+      @click="$emit('click', $event)"
+    >
+      <ButtonCancel />
+    </ButtonPlain>
+    <CookieImg class="cookie-img" />
+    <div class="info">
+      Superhero.com respects your privacy. Third-party tracking cookies are blocked by default.
+      In order to play content such as video or audio
+      you have to allow third-party cookies by content provider.
+    </div>
+    <div>
+      <ButtonPlain
+        class="cookies-button"
+        @click="allow"
+      >
+        Allow {{ scope }}
+      </ButtonPlain>
+      <ButtonPlain
+        v-if="address"
+        class="cookies-button"
+        :to="{ name: 'user-profile', params: { address } }"
+      >
+        Customize policy
+      </ButtonPlain>
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapState } from 'vuex';
+import backendAuthMixin from '../utils/backendAuthMixin';
+import ButtonPlain from './ButtonPlain.vue';
+import CookieImg from '../assets/cookieImg.svg?icon-component';
+import ButtonCancel from '../assets/buttonCancel.svg?icon-component';
+
+export default {
+  components: { ButtonPlain, CookieImg, ButtonCancel },
+  mixins: [backendAuthMixin()],
+  props: { scope: { type: String, required: true } },
+  computed: mapState(['address']),
+  methods: {
+    async allow() {
+      if (this.address) {
+        await this.backendAuth(`setCookies${this.scope}`, {
+          scope: this.scope,
+          status: 'ALLOWED',
+        });
+      }
+      this.$store.commit('setCookiesListEntry', {
+        scope: this.scope,
+        status: 'ALLOWED',
+      });
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.cookies-dialog {
+  position: absolute;
+  z-index: 9;
+  left: -4%;
+  background-color: $buttons_background;
+  border: 1px solid $article_content_color;
+  border-radius: 0.5rem;
+  width: 108%;
+  padding: 1rem;
+  cursor: default;
+  box-shadow: 4px 4px 4px 0 rgba(0, 0, 0, 0.3);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  .button-cancel {
+    position: absolute;
+    right: 0;
+    top: 0;
+
+    svg path {
+      fill: $tip_note_color;
+    }
```

Comment:
```suggestion
    color: $tip_note_color;
```
And to use `currentColor` in SVG